R/pkg/R/client.R

```diff
@@ -102,10 +102,17 @@ checkJavaVersion <- function() {
     javaVersionNum <- as.integer(versions[1])
   }
   if (javaVersionNum < minJavaVersion || javaVersionNum >= maxJavaVersion) {
-    stop(paste0("Java version, greater than or equal to ", minJavaVersion,
-                " and less than ", maxJavaVersion,
-                ", is required for this package; found version: ",
-                javaVersionStr))
+    stop(
```

Comment:
please advise on coding style for a case like this

Reply:
Let's just don't change this line :-). We can just remove `paste0` I guess.